packages/widget-notes/src/Widget.tsx

```diff
@@ -17,7 +24,10 @@ import '../css/react-splitter-layout.css'
 import NoteContext, { NoteProvider } from './Context'
 import NoteEditor from './components/Editor'
 import NoteListItem from './components/ListItem'
-import { Note } from './types'
+import { Events, Note } from './types'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faCloud } from '@fortawesome/free-solid-svg-icons'
+import FeatureInterestDialog from '@vscode-marquee/utils/src/components/FeatureInterestDialog'
```

Comment:
This won't work when we release the extension because all build files are in a dist directory. Better export the dialog in the utils package like all other components and do this:
```suggestion
import { FeatureInterestDialog } from '@vscode-marquee/utils'
```